examples/btc-to-evm-fungible-transfer/.env.sample

```diff
@@ -0,0 +1,13 @@
+SYGMA_ENV=testnet
+BLOCKSTREAM_URL="your blockstream url"
+DESTINATION_ADDRESS="your evm destination address"
+DESTINATION_DOMAIN_CHAIN_ID="destination domain chain id"
+RESOURCE_ID="resource id"
+SOURCE_DOMAIN_CAIPID="source domain caip id"
+EXPLORER_URL="your bitcoin explorer url"
+MNEMONIC="your 12 or 24 mnemonic"
+DERIVATION_PATH="your derivation path"
+UTXO_TX_ID="your utxo tx id"
```

Comment:
is there a possibility of not having to specify these values? can the SDK auto-select this

Reply:
for this SDK would need to connect to some external source of data to fetch the UTXO information from an address

Reply:
what happens if an invalid utxo id is provided by the user?

Reply:
it wont be able to construct the PSBT but I should test this. Thats a good point.

Reply:
hey @saadjhk confirming this: rather than not being able to construct the PSBT, which you can, you are not going to be able to sign in so I don't think -at first glance- there is an issue here providing wrong utxo data. We return the PSBT for the user to sign. If the utxo data doesn't belong to the signer the user is trying to use, upon calling the signing method on the PSBT to extract the tx and broadcast it, is going to throw